### PR TITLE
Tools: Tradheli SITL Params update

### PR DESCRIPTION
**Pull request by rbeall**
Enabled EKF2 else position doesn't properly come in and SITL fails to pre-arm.  Slightly adjusted tail tune to increase performance.

- EK2_ENABLE 1.0000
- ATC_RAT_YAW_P 0.5
- ATC_RAT_YAW_I 0.3

### CODE REVIEW

File: Tools/autotest/default_params/copter-heli.parm [hunk 1]
```diff
@@ -8,13 +8,14 @@ COMPASS_OFS3_Z 1.0000
...
 COMPASS_OFS_Y -21.8262
 COMPASS_OFS_Z -21.8422
+EK2_ENABLE 1.0000
 FLTMODE1 7.0000
 FLTMODE2 9.0000
 FLTMODE3 6.0000
 FLTMODE4 3.0000
 FLTMODE5 5.0000
 FRAME_CLASS 6
-# we need small INS_ACC offsets so INS is recognised as being calibrated
```
Comment (rmackay9) [inline Tools/autotest/default_params/copter-heli.parm:17]: this is just a difference between English and US spelling I think..

File: Tools/autotest/default_params/copter-heli.parm [hunk 2]
```diff
@@ -8,13 +8,14 @@ COMPASS_OFS3_Z 1.0000
 COMPASS_OFS_X 2.8749
 COMPASS_OFS_Y -21.8262
 COMPASS_OFS_Z -21.8422
+EK2_ENABLE 1.0000
```
Comment (rmackay9) [inline Tools/autotest/default_params/copter-heli.parm:11]: master is using EKF3 so I'm not sure we should be enabling EKF2

**Comment by rmackay9** [comment]
It's up to @bnsgeyer but I think the first commit should be removed but the second could be merged.

**Comment by peterbarker** [comment]
Concur we don't want EK2 in these parameters.

@rbeall why did you include that change?

**Comment by bnsgeyer** [comment]
@peterbarker  I had spoken with @rbeall regarding that.  I asked him to reset his parameters and try SITL again.  I haven't heard back whether that worked for him.  I think the only change that I plan to incorporate is the trim servo value for Servo4.  I have been busy with other things but will get this cleaned up and merged.

File: Tools/autotest/default_params/copter-heli.parm (hunk 2 above)
Comment (magicrub) [reply Tools/autotest/default_params/copter-heli.parm:11]: I'm definitely in favor of keeping it at EKF3.. unless this is intended to test both somehow?

**Comment by magicrub** [comment]
@bnsgeyer @rbeall  what's the status on this? Do we want to keep the PID changes or shall we remove everything except SERVO4_TRIM as Bill mentioned? We would also need to rebase to make sure it jives with current master CI tests.

**Comment by rbeall** [comment]
No Longer Applicable.  Will revisit later if required.  Bill's new auto tune should probably be run against this either way at some point and gains updated.

**Pull request closed without merging**